test/e2e/network/netpol/kubemanager.go

```diff
@@ -252,14 +264,15 @@ func (k *kubeManager) deleteNamespaces(namespaces []string) error {
 	return nil
 }
 
-// waitForHTTPServers waits for all webservers to be up, on all protocols, and then validates them using the same probe logic as the rest of the suite.
-func (k *kubeManager) waitForHTTPServers(model *Model) error {
+// waitForHTTPServers waits for all webservers to be up, on all protocols sent in the input,  and then validates them using the same probe logic as the rest of the suite.
+func (k *kubeManager) waitForHTTPServers(model *Model, protocols []v1.Protocol) error {
```

Comment:
Protocols should just become part of `Model` -- that's the point of Model, to track information like this.

It shouldn't be passed separately.